todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskPresenter.java

```diff
@@ -73,7 +76,7 @@ void setupListeners() {
 
     @Override
     public void start() {
-        if (!isNewTask()) {
+        if (!isNewTask() && mIsDataMissing) {
```

Comment:
I don't think this is what you think it is. It's to fix a subtle bug that can happen if in a config change the data is not ready yet. It was added to todo-mvp a long time ago.